app/javascript/gobierto_plans/modules/plan_types_controller.js

```diff
@@ -81,6 +81,24 @@ window.GobiertoPlans.PlanTypesController = (function() {
                   Vue.set(this.model, "children", json)
                   this.$emit("toggle");
                   this.isOpen = !this.isOpen;
+
+
+                  json.forEach(element => {
```

Comment:
cc/ @ferblape @amiedes @entantoencuanto 

This snippet triggers an event called `gobierto-plans-mount-plugin` to inform the `tablerow_controller` that some div element has to to be mounted.
If there's no plugins, nothing happens. If so, is the plugin (tablerow_controller) who is listening such event and doing the stuff

Reply:
To avoid unnecessary calls, I got rid of `forEach`, moving this snippet to the component `table-view` and the root component as well